ush/get_platform.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/sh
```

Comment:
This script does not appear to actually be used anywhere, is this actually needed?

Reply:
If it's not needed in f/c-c then I will remove it from this PR.

Reply:
The load_fv3gfs_modules script uses it on the CROW side. I think Raffaele added it. I think it's a good helper script to have to stop repeating the logic in other scripts, we just need to go through and use it more extensively.

Reply:
I'm not saying it's not a good helper script, but if it's not being used, it's just extra and is not related to this PR and therefore should not be included in this PR.